tests/dummy/app/styles/tables.scss

```diff
@@ -4,6 +4,19 @@
   }
 }
 
+.demo-options-group {
+  display: flex;
+
+  & *:first-child {
+    text-align: right;
+    flex-basis: 40%;
+  }
+
+  & > * + * {
```

Comment:
😬 :mindblown:

Reply:
Given `<a><b /><c /></a>` this only matches `c`?

Reply:
Haha, yeah. It selects "an element that follows an element". I discovered this recently and am probably over-using it... :) It's a cleaner alternative to a rule that sets some padding and then a ":first-child" or ":last-child" that removes the padding.

https://alistapart.com/article/axiomatic-css-and-lobotomized-owls/